tests/SpecTests/Operation.php

```diff
@@ -411,12 +407,8 @@ private function executeForCollection(Collection $collection, Context $context):
                 return $collection->listIndexes($args);
 
             case 'mapReduce':
-                return $collection->mapReduce(
-                    $args['map'],
-                    $args['reduce'],
-                    $args['out'],
-                    array_diff_key($args, ['map' => 1, 'reduce' => 1, 'out' => 1]),
-                );
+                Assert::markTestSkipped('mapReduce is not supported');
+                break;
```

Comment:
I suppose there's no harm in it, but is this `break` actually necessary? Can we rely on Psalm to assert that this line is unreachable?

Reply:
`phpcs` was complaining about this. While psalm is able to infer that this won't be reached due to a `@return never`, `phpcs` doesn't and can only use a `return` within a case to infer that a `break` statement is not needed.